code/controllers/subsystem/processing/day_night.dm

```diff
@@ -0,0 +1,129 @@
+
+
+/**
+ * STATION TIMES ARE 24 HR FORMAT
+ */
+
+SUBSYSTEM_DEF(day_night)
+	name = "Day/Night Cycle"
+	wait = 6 SECONDS // Every minute, the clock moves forward 1 minutes
+	init_order = INIT_ORDER_DAY_NIGHT
+	/// The current hour
+	var/current_hour = 0
+	/// The current minute
+	var/current_minute = 0
+	/// A list of all currently loaded controllers to be handled
+	var/list/cached_controllers = list()
```

Comment:
controller? as in the master type of anything that goes in the mc tab?

Reply:
No, each loaded day night cycle controller, there can be more than one if say there's a planet ruin or something.

Reply:
could you use types here? so list/datum/controller_type?